scripts/local/dev-component-library-install.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+GREEN='\033[1;32m'
+ENDCOLOR='\033[0m'
+
+read -p "Which branch? " BRANCH
```

Comment:
I'd add the repo name here, i.e. "Which branch from the component-library-twig repo" or something like that so it's clearer which repo you are asking for.

Reply:
Done